ophyd/device.py

```diff
@@ -341,10 +349,16 @@ def describe(self):
 
     def stage(self):
         "Prepare the device to be triggered."
-        if self._staged:
-            raise RuntimeError("Device is already stage. Unstage it first.")
-        self._staged = True
+        if self._staged == Staged.no:
+            pass  # to short-circuit checking individual cases
```

Comment:
My response got lost in the ether of intermittent cell service:

I wouldn't call it placeholder. It optimizes the case where everything is normal (making one comparison instead of two) at the cost of slowing down the case where we have to raise an error (making two or three comparisons instead of one or two).